cluster-scope/base/rbac.authorization.k8s.io/clusterroles/katalog-editor/clusterrole.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: katalog-editor
+rules:
+- apiGroups:
+  - katalog.fybrik.io
+  resources:
+  - assets
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+  - deletecollection
+- apiGroups:
```

Comment:
I think this role might be too much, since it let's the user get secrets from all over the cluster.
@HumairAK @tumido